framework/assets/yii.gridView.js

```diff
@@ -123,7 +123,7 @@
             var $form = $('<form/>', {
                 action: url,
                 method: 'get',
-                class: 'gridview-filter-form',
+                'class': 'gridview-filter-form',
```

Comment:
I see. So this is a recent addition to ECMA.

I suppose all option keys should be quoted then for the consistency, not just `class`. And same fix should be applied to other places too.